README.md

```diff
@@ -109,9 +120,33 @@ auto-round-fast \
     --group_size 128 \
     --disable_eval 
   ```
+<br>
+
+#### Auto-Round Recipes Results
+In general, it is recommended to use the auto-round default mode. When resources or quantization time are a priority, the auto-round-light mode can be preferred for models larger than 3B. Below are the quantization results for models ranging from 3B to 72B as a reference(with torch compile enabled).
+
+- Accuracy Results
+
+| Config\Model | Qwen2.5-7B-Instruct | llama3.1-8b-instruct | falcon3-10b | OLMo-2-1124-7B-Instruct | Qwen2.5-72B-Instruct |
+|:--------------:|:---------------------:|:----------------------:|:-------------:|:-------------------------:|:----------------------:|
+| 16bits       | 0.6470              | 0.6212               | 0.6151      | 0.6268                  | 0.7229               |
+| Best         | 0.6426              | **0.6115**               | **0.6092**      | **0.6295**                  | 0.7242               |
+| Default      | 0.6441              | 0.6106               | 0.6080      | 0.6253                  | **0.7252**               |
+| Light        | **0.6453**              | 0.6111               | 0.6063      | 0.6261                  | 0.7243               |
+
+- Time Costs
+
+| Config\Model | Qwen2.5-7B-Instruct | llama3.1-8b-instruct | falcon3-10b | OLMo-2-1124-7B-Instruct | Qwen2.5-72B-Instruct |
+|:--------------|---------------------:|----------------------:|-------------:|-------------------------:|----------------------:|
+| Best         | 3425                | 3754                 | 4840        | 3360                    | 33984                |
+| Default      | 739                 | 757                  | 1046        | 704                     | 7076                 |
+| Light        | 306                 | 255                  | 410         | 311                     | 2273                 |
```

Comment:
retest all the models of  7B and 8B, the time cost is not  reasonable,  7B needs 300s while 8B only needs 255s